### ISSUE TITLE
`CompressionUtils` only once

### ISSUE THREAD

**Issue by drcrallen** [apache/incubator-druid#4045]
`CompressionUtils` exists in both `java-util` and `api`. This ask is that there be just one class called `CompressionUtils`.

**Comment by leventov** [comment]
In general, `java-util` and `bytebuffer-collections` modules should be dissipated in other modules, mainly merged into `druid-common`
